Source/Eto/Drawing/PaddingConverter.cs

```diff
@@ -12,6 +12,11 @@ namespace Eto.Drawing
 	public class PaddingConverter : TypeConverter
 	{
 		/// <summary>
+		/// The character to split up the string which will be converted
+		/// </summary>
+		private readonly string[] StringSplitter = new string[1] { "," };
```

Comment:
I'd prever `char[]` instead of `string[]` but this is getting..... ugh..... funny polishing :D